pkg/network/infraconfigurators/masquerade_test.go

```diff
@@ -392,8 +413,12 @@ func mockNFTablesFrontend(handler *netdriver.MockNetworkHandler, proto iptables.
 	handler.EXPECT().NftablesAppendRule(proto, "nat", "prerouting", "iifname", "eth0", "counter", "jump", "KUBEVIRT_PREINBOUND").Return(nil)
 	handler.EXPECT().NftablesAppendRule(proto, "nat", "postrouting", "oifname", "k6t-eth0", "counter", "jump", "KUBEVIRT_POSTINBOUND").Return(nil)
 
-	for _, chain := range []string{"output", "KUBEVIRT_POSTINBOUND"} {
-		handler.EXPECT().NftablesAppendRule(proto, "nat", chain, "tcp", "dport", fmt.Sprintf("{ %s }", strings.Join(PortsUsedByLiveMigration(), ", ")), nftIPString, "saddr", GetLoopbackAdrress(proto), "counter", "return").Return(nil)
+	skipPorts := portsUsedByLiveMigration(isMigrationOverSockets)
+
+	if len(skipPorts) > 0 {
```

Comment:
Limiting the scope of `skipPorts` may be useful to be clear it is not used later on.
`if skipPorts := portsUsedByLiveMigration(isMigrationOverSockets); len(skipPOrts) > 0 {`